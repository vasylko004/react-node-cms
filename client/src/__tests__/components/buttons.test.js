import React from 'react';
import renderer from 'react-test-renderer';
import Button from '../../components/buttons';

test("Buttons rendering tests", ()=>{
    let component = renderer.create(<Button classes="test" size="std" > Test </Button>)
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();

    component = renderer.create(<Button classes="test_large" size="large" > Test 2 </Button>)
    tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});