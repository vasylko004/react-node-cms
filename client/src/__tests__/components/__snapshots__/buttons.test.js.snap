// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Buttons rendering tests 1`] = `
<button
  className="waves-effect waves-lighttest btn"
>
   
   Test 
   
</button>
`;

exports[`Buttons rendering tests 2`] = `
<button
  className="waves-effect waves-lighttest_large btn-large"
>
   
   Test 2 
   
</button>
`;
